tests/gtest/avifcolrconverttest.cc

```diff
@@ -15,36 +15,38 @@ namespace {
 // Used to pass the data folder path to the GoogleTest suites.
 const char* data_path = nullptr;
 
-void ExpectMatrixNear(const double actual[3][3],
+void ExpectMatrixNear(const double actual[9],
                       const std::array<std::array<double, 3>, 3>& expected,
                       const double epsilon) {
-  EXPECT_NEAR(actual[0][0], expected[0][0], epsilon);
-  EXPECT_NEAR(actual[0][1], expected[0][1], epsilon);
-  EXPECT_NEAR(actual[0][2], expected[0][2], epsilon);
-  EXPECT_NEAR(actual[1][0], expected[1][0], epsilon);
-  EXPECT_NEAR(actual[1][1], expected[1][1], epsilon);
-  EXPECT_NEAR(actual[1][2], expected[1][2], epsilon);
-  EXPECT_NEAR(actual[2][0], expected[2][0], epsilon);
-  EXPECT_NEAR(actual[2][1], expected[2][1], epsilon);
-  EXPECT_NEAR(actual[2][2], expected[2][2], epsilon);
+  EXPECT_NEAR(actual[0], expected[0][0], epsilon);
+  EXPECT_NEAR(actual[1], expected[0][1], epsilon);
+  EXPECT_NEAR(actual[2], expected[0][2], epsilon);
+  EXPECT_NEAR(actual[3], expected[1][0], epsilon);
+  EXPECT_NEAR(actual[4], expected[1][1], epsilon);
+  EXPECT_NEAR(actual[5], expected[1][2], epsilon);
+  EXPECT_NEAR(actual[6], expected[2][0], epsilon);
+  EXPECT_NEAR(actual[7], expected[2][1], epsilon);
+  EXPECT_NEAR(actual[8], expected[2][2], epsilon);
 }
 
 TEST(RgbToXyzD50Matrix, GoldenValues) {
-  double coeffs[3][3];
+  double coeffs[9];
   ASSERT_TRUE(avifColorPrimariesComputeRGBToXYZD50Matrix(
       AVIF_COLOR_PRIMARIES_BT709, coeffs));
   // Golden values from
   // http://brucelindbloom.com/index.html?Eqn_RGB_XYZ_Matrix.html
   const double kEpsilon = 0.00015;
   ExpectMatrixNear(coeffs,
-                   {{{0.4360747, 0.3850649, 0.1430804},
-                     {0.2225045, 0.7168786, 0.0606169},
-                     {0.0139322, 0.0971045, 0.7141733}}},
+                   {{
+                       0.4360747, 0.3850649, 0.1430804,  // row 0
+                       0.2225045, 0.7168786, 0.0606169,  // row 1
+                       0.0139322, 0.0971045, 0.7141733   // row 2
```

Comment:
If the original still works, the original code seems to be a better initializer for `std::array<std::array<double, 3>, 3>` (see line 19).

This comment also applies to the second arguments for the `ExpectMatrixNear()` calls below.